modules/apps/translation/translation-google-cloud-translator/src/main/java/com/liferay/translation/google/cloud/translator/internal/configuration/persistence/listener/GoogleCloudTranslatorConfigurationModelListener.java

```diff
@@ -0,0 +1,83 @@
+/**
```

Comment:
When introducing a new class that implements an interface
        take into account:

* The class name should end with the interface name
* The package of the class matches the package of the interface



Refer to documentation: